github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala

```diff
@@ -756,4 +757,145 @@ class ApiSpec
       None)
     response should be('left)
   }
+
+  "Issues >> List" should "return the expected issues when a valid owner/repo is provided" in {
+    val response =
+      issues.list(accessToken, headerUserAgent, validRepoOwner, validRepoName)
+    response should be('right)
+
+    response.toOption map { r ⇒
+      r.result.nonEmpty shouldBe true
+      r.statusCode shouldBe okStatusCode
+    }
+  }
+
+  it should "return an error if invalid data is provided" in {
+    val response =
+      issues.list(accessToken, headerUserAgent, validRepoOwner, invalidRepoName)
+    response should be('left)
+  }
+
+  it should "return an error if no tokens are provided" in {
+    val response =
+      issues.list(None, headerUserAgent, validRepoOwner, validRepoName)
+    response should be('left)
+  }
+
+  "Issues >> Create" should "return the created issue if valid data is provided" in {
+    val response = issues.create(
+      accessToken,
+      headerUserAgent,
+      validRepoOwner,
+      validRepoName,
+      validIssueTitle,
+      validIssueBody,
+      None,
+      List.empty,
+      List.empty)
+    response should be('right)
+
+    response.toOption map { r =>
+      r.statusCode shouldBe createdStatusCode
+    }
+  }
+
+  it should "return an error if invalid data is provided" in {
+    val response = issues.create(
+      accessToken,
+      headerUserAgent,
+      validRepoOwner,
+      invalidRepoName,
+      validIssueTitle,
+      validIssueBody,
+      None,
+      List.empty,
+      List.empty)
+    response should be('left)
+  }
+
+  it should "return an error if no tokens are provided" in {
+    val response = issues.create(
+      None,
+      headerUserAgent,
+      validRepoOwner,
+      validRepoName,
+      validIssueTitle,
+      validIssueBody,
+      None,
+      List.empty,
+      List.empty)
+    response should be('left)
+  }
+
+  "Issues >> Edit" should "return the edited issue if valid data is provided" ignore {
```

Comment:
You guys seem to have used post for patch request for the git data API: https://github.com/47deg/github4s/blob/master/github4s/jvm/src/test/scala/github4s/utils/MockGithubApiServer.scala#L328-L348.

So I'm assuming there is something wrong with how mock server deals with patch?